src/components/common.js

```diff
@@ -52,7 +52,7 @@ const styles = {
   }
 }
 
-export const Clickable = ({ href, as = (!!href ? 'a' : 'div'), disabled, tooltip, tooltipSide, tooltipDelay, onClick, children, ...props }) => {
+export const Clickable = ({ href, as = (!!href ? 'a' : 'div'), disabled, tooltip, tooltipSide, tooltipDelay, useTooltipAsLabel, onClick, children, ...props }) => {
```

Comment:
I snuck this in here so I could start to use the attribute without waiting for #2525 to get merged and without causing DOM warnings. May possible cause a merge conflict once that branch is merged, but it'll be easy to resolve.